docs/privacyidea.md

```diff
@@ -97,8 +97,10 @@ You need to add the authentication source 'privacyidea' to
      * Set preferredTokenType to your favourite token type.
      * If the choosen token is triggered, it will be used to authenticate directly
      * without having to press the button for the type.
+     * Possible values are: push, webauthn or u2f.
```

Comment:
@nilsbehlen  Does this correcpond to the server defined client modes? This would be

* interactive (default)
* webauthn or u2f
* poll (which is the generic information without naming it push, see https://privacyidea.readthedocs.io/en/latest/tokens/authentication_modes.html)

Reply:
It's the names of the modes how they currently implemented, which is "mapped" from token type. When a mode is contained in the response, we can change the implentation to that.

Reply:
right, the server will respond with "poll" in case of push tokens.